src/App.tsx

```diff
@@ -292,9 +428,15 @@ class Numpad extends React.Component<NumpadProps, {}> {
 
     public renderNumButton(i: SquareValue) {
         return <Button
-            onClick={() => {
+            onClick={(e) => {
                 this.props.numpadMode === "normal" ? this.props.onClickNormal(i) : this.props.onClickCorner(i);
             }}
+            onMouseDown={(event) => {
```

Comment:
On second thought, I think you want `stopPropagation` instead of `preventDefault`, right?
https://developer.mozilla.org/en-US/docs/Web/API/Event/stopPropagation